### PR TITLE
📝 add readme missing step

### LINKED ISSUE #42: Add missing step in readme
In the current version it misses `protostar install` before the compilation topic

### PR DESCRIPTION
**Pull request by Bal7hazar**
Fix #42 

- [x] add protostar install step

### CODE REVIEW

File: protostar.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ["protostar.config"]
-protostar_version = "0.3.1"
+protostar_version = "0.3.2"
```
Comment (cloudvenger) [inline protostar.toml:2]: careful, it also in the other PR, you will have conflict during the merge. 
I will validate this PR (wating to answer to other), so I recommand to merge with main repo before PR in other PR.

**Review by cloudvenger — APPROVED** [review]
careful, it also in the other PR, you will have conflict during the merge.
I will validate this PR (wating to answer to other), so I recommand to merge with main repo before PR in other PR.

**Comment by cloudvenger** [comment]
During the test: [ERROR] Protostar v0.3.2 is not compatible with provided protostar.toml.

**Comment by cloudvenger** [comment]
In the file "protostar,toml", you need to change the target value with:
```
["protostar.config"]
protostar_version = "0.3.2"
```

**Comment by Bal7hazar** [comment]
If I understand it well:
- `pull_request_target` is the CI of the target branch without modification (so with no upgrade of protostar version in the toml file), basically it fails because of the new protostar version
- `pull_request` is the merged version (including the changes)

In addition, I'm a bit confused about your notes @cloudvenger:

- You suggest here to upgrade the protostar version:

> In the file "protostar,toml", you need to change the target value with:
> 
> ```
> ["protostar.config"]
> protostar_version = "0.3.2"
> ```

And there you warn me about this same update (which is already done in my branch btw :sweat_smile:)

> careful, it also in the other PR, you will have conflict during the merge. I will validate this PR (wating to answer to other), so I recommand to merge with main repo before PR in other PR.

**Comment by cloudvenger** [comment]
To not blocking, I will accepted the PR but then I will check what we need to expected from `pull_request_target` and `pull_request_target`. 

Sorry if I was not clear, to clarify:
* My point was to ask the modification of "protostar_version = "0.3.2" only in one PR, not in two PR distinct.

**Review by cloudvenger — APPROVED** [review]

**Pull request merged**